source/server/admin/stats_html_render.h

```diff
@@ -10,16 +10,20 @@ class StatsHtmlRender : public StatsTextRender {
   StatsHtmlRender(Http::ResponseHeaderMap& response_headers, Buffer::Instance& response,
                   const StatsParams& params);
 
+  void dynamicParams(Buffer::Instance& response);
+
   void noStats(Buffer::Instance&, absl::string_view types) override;
   void generate(Buffer::Instance& response, const std::string& name,
                 const std::string& value) override;
+  /*
```

Comment:
stale code?

Reply:
actually turns out that code is needed to get gcc to build in CI, otherwise gcc gives warnings about hidden virtual functions. I uncommented and added comments.